app/controllers/assignments_controller.rb

```diff
@@ -92,7 +95,35 @@ def create
       @assignment.available = @assignment.due_date
     end
 
-    if @assignment.save
+    # Options are:
+    # None -- create a (unique, ignored) teamset
+    # New -- create a new teamset with initially no teams
+    # Use -- share an existing teamset with another assignment
+    # Copy -- duplicate an existing teamset
+    
+    @assignment.team_subs = (params["teamset"] != "none")
```

Comment:
I...don't understand how it could be true for longer in my version.  Yes, my code isn't transactional yet, and it probably should be.  That's a bigger task that's orthogonal to this one, though.